app/src/main/java/com/codingblocks/cbonlineapp/mycourse/MyCourseViewModel.kt

```diff
@@ -45,7 +48,11 @@ class MyCourseViewModel(
     var name by savedStateValue<String>(handle, COURSE_NAME)
     var runId by savedStateValue<String>(handle, RUN_ID)
 
+    val userName = PreferenceHelper.USER_NAME
+    val userId = PreferenceHelper.USER_ID
```

Comment:
Not to be done like this 
See implementation in other classes

Reply:
Also, id would be sufficient rest of the information can be fetched from API

Reply:
Can you direct me to one instance or class where i can find? I tried searching other viewmodels but didn't find any such implementation